.github/workflows/check-licenses.yml

```diff
@@ -33,20 +33,11 @@ jobs:
         with:
           repository: eclipse-velocitas/license-check
           path: .github/actions/license-check
-          ref: main
+          ref: v1.2.0
 
       - name: Run License Checker
-        id: checker
-        continue-on-error: true
         uses: ./.github/actions/license-check
         with:
           config-file-path: ./.licensechecker.yml
           fail-on-violation: false
           notice-file-name: "NOTICE-3RD-PARTY-CONTENT"
```

Comment:
Optional: in the action.yaml we could change the. default file name to NOTICE-3RD-PARTY-CONTENT instead of NOTICE-GENERATED and just let the workflow uses the default :)

Reply:
I prefer to have it explicit in the repos.